src/services/requests.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+export const instance = axios.create({
+  baseURL: import.meta.env.BASE_URL,
+});
+
+instance.interceptors.request.use((config: AxiosRequestConfig): any => {
+  const accessToken = localStorage.getItem('accessToken');
+
+  if (accessToken && config.headers) {
+    config.headers['Authorization'] = accessToken;
+  }
+
+  return config;
+});
+
+axios.interceptors.response.use(
+  (response) => {
+    return response;
+  },
+  (error) => {
+    if (error.response.status === 401) {
+      // Refresh token
+    }
+    return error;
+  },
+);
```

Comment:
Do I correctly mean this logic? If the term of JWTtoken has ended, we will have 401 error and then we will replace this token to refresh token?

Reply:
Yes, when the expireTime of JWT token end, we will add here logic for refresh token